<launch>

  <!--Launch Turtlebot-->
  <include file="$(find turtlebot_bringup)/launch/minimal.launch"/>

  <!--Launch Realsense Camera-->
  <!--<include file="$(find realsense2_camera)/launch/rs_rgbd.launch"/>-->
  <node pkg="cv_camera" type="cv_camera_node" name="cv_camera">
    <param name="device_id" value="0"/>
  </node>

  <!--Joy node for xbox 360-->
  <node pkg="joy" type="joy_node" name="joystick"/>

  <!--Custom Joy commander-->
  <node pkg="deep_turtle" type="custom_joy_cmd.py" name="joy_command" output="screen">
    <param name="joy_velocity_max" value="1.0"/>
    <param name="joy_omega_max" value="1."/>
    <param name="constant_vel" value="0.3"/>
    <remap from="turtle_cmd" to="/mobile_base/commands/velocity"/>
    <remap from="auto_cmd" to="/auto_cmd_raw"/>
  </node>

  <!-- Predict from saved model-->
  <node pkg="deep_turtle" type="predict_continous_node.py" name="neural_controller" output="screen">
    <!--<param name="model_name" value="rope1/rope1"/>-->
    <remap from="rgb_image" to="/cv_camera/image_raw"/>
  </node>


</launch>
